README.md

```diff
@@ -1,7 +1,236 @@
+<!-- Code generated by "go generate" in github.com/kortschak/gospel; DO NOT EDIT. -->
 # `gospel` [![Build status](https://github.com/kortschak/gospel/workflows/Test/badge.svg)](https://github.com/kortschak/gospel/actions)
 
-The `gospel` program lints Go source code for misspellings in comments and strings. It uses hunspell to identify misspellings and only emits coloured output for visual inspection; don't use it in automated linting.
+The `gospel` program lints Go source files for misspellings in comments,
+strings and embedded files.
```

Comment:
nit: oxford comma

Reply:
I don't unless it's necessary to separate clauses that are ambiguous because of other conjunctions — you'll see that in the `.dic` files section — here it's not necessary.